package com.thinkinginjava.error_handling_with_exceptions.example.chapter12_6.ex12_6_3;

/**
 * page 260
 *   -- chapter_12 通过异常处理错误
 *
 *   12.6.3 异常链
 */